src/modules/home/communityBenefits/helper.ts

```diff
@@ -0,0 +1,22 @@
+type communityProps = {
+  name: string;
+  imageurl: string;
+};
+export const communityBenefitsList: communityProps[] = [
+  {
+    name: "Community Engagement",
+    imageurl: "./images/illustrations/group.svg",
+  },
+  {
+    name: "Learn and Share",
+    imageurl: "./images/illustrations/share.svg",
+  },
+  {
+    name: "Networking Hub",
+    imageurl: "./images/illustrations/networking.svg",
+  },
+  {
+    name: "Holistic Growth",
+    imageurl: "./images/illustrations/helping.svg",
+  },
+];
```

Comment:
Why is this repeted twice ? saw that on `src/components/layout/helper.ts` as well.